apps/frontend/src/components/meetings/InitialMeet.tsx

```
@@ -43,9 +41,10 @@
         }
 
         const meet: Meet | null = await fetchLatestMeet(session)
+        console.log("single meet: ", meet)
 
-        if (meet && meet._id) {
-          router.push(`/space/meetings/${meet._id}`)
+        if (meet && meet.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)